eng/pipelines/compliance.yml

```diff
@@ -27,11 +27,33 @@ jobs:
           targetArgument: "$(Build.SourcesDirectory)"
           result: "PoliCheck.xml"
 
+      - task: DownloadPipelineArtifact@2
+        displayName: "Download symbols - NonRTM artifact"
+        inputs:
+          source: specific
+          project: 'devdiv'
+          pipeline: NuGet.Client-Official
```

Comment:
As per [doc](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-pipeline-artifact?view=azure-devops#how-can-i-find-the-id-of-the-pipeline-i-want-to-download-an-artifact-from) pipeline value is ` the ID of the pipeline in the 'Pipeline variables'. The pipeline ID is the $(System.DefinitionId) variable. Since you have already tested the task with `NuGet.Client-Official` as pipeline value. We should be good.